Makefile

```diff
@@ -185,6 +185,7 @@ deploy-on-openshift: oc
        -p MIGRATION_PLANNER_URL=http://planner-agent-$${openshift_project}.apps.$${openshift_base_url}/api/migration-assessment \
        -p MIGRATION_PLANNER_UI_URL=http://planner-ui-$${openshift_project}.apps.$${openshift_base_url} \
        -p MIGRATION_PLANNER_IMAGE_URL=http://planner-image-$${openshift_project}.apps.$${openshift_base_url} \
+	   -p ISO_PV_ACCESS_MODE="ReadWriteOnce" \
```

Comment:
If we have a default value set the same, let's remove this? Same for the things below